awslimitchecker/tests/test_runner.py

```diff
@@ -614,6 +619,19 @@ def test_entry_no_service_name(self, capsys):
         assert excinfo.value.code == 6
         assert self.cls.service_name is None
 
+    def test_entry_no_service_name_skip_service(self, capsys):
```

Comment:
My question regarding tests was here.

Can we perform a better test than this?